channels/applicationinsights-channel-js/Tests/Unit/src/Sender.tests.ts

```diff
@@ -22,6 +22,7 @@ export class SenderTests extends AITestClass {
 
         if (this._sender && this._sender.isInitialized()) {
             this._sender.pause();
+            this._sender._buffer.clear();
```

Comment:
I realised while writing my test that session storage wasn't being cleared between tests 😅